internal/cluster/validations/validations.go

```diff
@@ -506,9 +525,20 @@ func ValidateClusterUpdateVIPAddresses(ipV6Supported bool, cluster *common.Clust
 	targetConfiguration.UserManagedNetworking = swag.Bool(false)
 	if params.UserManagedNetworking != nil {
 		if swag.BoolValue(params.UserManagedNetworking) {
-			err = ValidateVIPsWereNotSetUserManagedNetworking(cluster.APIVip, cluster.IngressVip,
-				cluster.APIVips, cluster.IngressVips, swag.BoolValue(cluster.VipDhcpAllocation))
-			if err != nil {
+
+			if swag.BoolValue(cluster.VipDhcpAllocation) { // override VIPs that were allocated via DHCP
+				params.APIVip = swag.String("")
+				params.IngressVip = swag.String("")
+				params.APIVips = []*models.APIVip{}
+				params.IngressVips = []*models.IngressVip{}
+			}
```

Comment:
Shouldn't this be done on validateVIPsWereNotSetUserManagedNetworking func?
I mean, in an 'else' branch on validations.go line 489.

Reply:
Oh actually maybe I'm missing something:) Is this part of the validation, or just an override of the params for later in the flow?:/

Reply:
This override is needed to clean up these VIPs for this specific scenario